modules/heart/heart.js

```diff
@@ -0,0 +1,64 @@
+import Component from "@pencil.js/component";
+
+/**
+ * Heart class
+ * @class
+ * @extends Component
+ */
+export default class Heart extends Component {
+    /**
+     * Heart constructor
+     * @param {PositionDefinition} positionDefinition - Position of the center of the heart
+     * @param {Number} radius - radius The distance from center of heart
+     * @param {ComponentOptions} [options] - Drawing options
+     */
+    constructor (positionDefinition, radius, options) {
+        super(positionDefinition, options);
+
+        if (radius > 0) {
+            this.radius = radius;
+        }
+        else {
+            this.radius = 1;
+        }
+    }
+
+    /**
+     * @inheritDoc
+     */
+    toJSON () {
+        const json = super.toJSON();
+        delete json.radius;
+        return json;
+    }
+
+    /**
+     * Draw the arc
+     * @param {Path2D} path - Drawing context
+     * @return {Heart} Itself
+     */
+    trace (path) {
+        path.moveTo(75 * this.radius, 40 * this.radius);
+        path.bezierCurveTo(75 * this.radius, 37 * this.radius, 70 * this.radius, 25 * this.radius,
```

Comment:
The double line was to avoid the lint. But I saw that you decided to ignore somewhere else in the project.
I can put them back in one line if you want.